packages/components/psammead-podcast-promo/src/index.test.jsx

```diff
@@ -85,28 +85,13 @@ describe('Podcast Promo', () => {
   });
 
   describe('card-call-to-action', () => {
-    assertTypeOfElement(PodcastPromo.Card.CallToAction, 'span');
+    assertTypeOfElement(PodcastPromo.Card.CallToAction, 'p');
     shouldMatchSnapshot(
       'should match snapshot',
       <PodcastPromo.Card.CallToAction script={scripts.latin} service="russian">
```

Comment:
should russian be hard coded?

Reply:
We need to hard code _some_ service value here. It probably doesn't make sense to have latin script and English text with the Russian service. I'll address this as part of https://github.com/bbc/psammead/issues/4123